package.json

```diff
@@ -35,7 +35,7 @@
     "lint-staged": "^14.0.1",
     "prettier": "^2.8.8",
     "ts-node": "^10.9.1",
-    "typechain": "^8.1.1"
+    "typechain": "^8.3.1"
```

Comment:
will the autogenerated types change? if so, maybe a follow up to re-create the types might be benefitial

Reply:
Autogenerated types aren't committed into the repository so there's nothing to recreate. This is actually just a version bump that didn't result in any substantive change (I was hopeful that this was going to include custom error types, but it seems this is [still yet to be done in typechain](https://github.com/dethcrypto/TypeChain/issues/667).